cmd/get_test.go

```diff
@@ -50,8 +50,8 @@ func TestGetComplete(t *testing.T) {
 			continue
 		}
 
-		if c.path != opt.Path {
-			t.Errorf("case %d, opt.Path not set correctly. Expected: '%s', Got: '%s'", i, c.path, opt.Path)
+		if c.selector != opt.Selector {
+			t.Errorf("case %d, opt.Path not set correctly. Expected: '%s', Got: '%s'", i, c.selector, opt.Selector)
```

Comment:
path var name has changed: `case %d opt.Selector not set correctly...`

Reply:
Good catch, done.